### PR TITLE
fix: OHIF-37 / OHIF-160 - Thumbnail double click + Active thumbnail

### PR DESCRIPTION
**Pull request by rodrigolabs**
**OHIF-37**
**OHIF-160**

### Description

- doubleClick thumbnail to load the series in the active viewport
- Series thumbnail active state -- also fixed hover/focus styles for Thumbnails.
- Thumbnail will be highlighted as actived based on the viewport current display set

### Preview

Jun-30-2020 17-56-47

### PR Checklist

- [ ] Brief description of changes
- [ ] Links to any relevant issues
- [ ] Required status checks are passing
- [ ] User cases if changes impact the user's experience
- [ ] `@mention` a maintainer to request a review

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

File: platform/ui/src/components/Thumbnail/Thumbnail.jsx [hunk 1]
```diff
@@ -30,52 +31,57 @@ const Thumbnail = ({
...
+
   return (
     <div
-      ref={drag}
+      onFocus={() => blurHandlerListener(thumbnailElement)}
+      ref={thumbnailElement}
       className={classnames(
         className,
-        'flex flex-col flex-1 px-3 mb-8 cursor-pointer outline-none'
+        'flex flex-col flex-1 px-3 mb-8 cursor-pointer outline-none group'
```
Comment (dannyrb) [inline platform/ui/src/components/Thumbnail/Thumbnail.jsx:42]: What does `group` do?

File: platform/ui/src/components/ThumbnailNoImage/ThumbnailNoImage.jsx [hunk 2]
```diff
@@ -20,37 +23,57 @@ const ThumbnailNoImage = ({
...
+      onDoubleClick={onClick}
       role="button"
       tabIndex="0"
     >
-      <div className="flex flex-col flex-1">
-        <div className="flex flex-row items-center flex-1 mb-2">
-          <Icon name="list-bullets" className="w-12 text-secondary-light" />
-          <div className="px-3 mr-4 text-lg text-white rounded-sm bg-primary-main">
-            {modality}
+      <div ref={drag}>
```
Comment (dannyrb) [inline platform/ui/src/components/ThumbnailNoImage/ThumbnailNoImage.jsx:41]: I could have sworn these were already draggable. What changed?

File: platform/ui/src/components/ThumbnailNoImage/ThumbnailNoImage.jsx [hunk 3]
```diff
@@ -20,37 +23,57 @@ const ThumbnailNoImage = ({
...
+   * Data the thumbnail should expose to a receiving drop target. Use a matching
+   * `dragData.type` to identify which targets can receive this draggable item.
+   * If this is not set, drag-n-drop will be disabled for this thumbnail.
+   *
+   * Ref: https://react-dnd.github.io/react-dnd/docs/api/use-drag#specification-object-members
+   */
+  dragData: PropTypes.shape({
+    /** Must match the "type" a dropTarget expects */
+    type: PropTypes.string.isRequired,
+  }),
```
Comment (dannyrb) [inline platform/ui/src/components/ThumbnailNoImage/ThumbnailNoImage.jsx:71]: Oops. Didn't realize I'd forgotten dragData propTypes. Thanks!

File: platform/ui/tailwind.config.js [hunk 4]
```diff
@@ -721,7 +721,7 @@ module.exports = {
     backgroundRepeat: ['responsive'],
     backgroundSize: ['responsive'],
     borderCollapse: ['responsive'],
-    borderColor: ['responsive', 'hover', 'focus', 'active'],
+    borderColor: ['responsive', 'hover', 'focus', 'active', 'group-focus'],
```
Comment (dannyrb) [inline platform/ui/tailwind.config.js:724]: `group-focus` 👀

File: platform/ui/src/components/StudyBrowser/StudyBrowser.jsx [hunk 5]
```diff
@@ -121,6 +111,7 @@ StudyBrowser.propTypes = {
   onClickUntrack: PropTypes.func,
   activeTabName: PropTypes.string.isRequired,
   expandedStudyInstanceUIDs: PropTypes.arrayOf(PropTypes.string).isRequired,
+  currentDisplaySetInstanceUID: PropTypes.string.isRequired,
```
Comment (dannyrb) [inline platform/ui/src/components/StudyBrowser/StudyBrowser.jsx:114]: This component has so much prop drilling 🙃

File: platform/ui/tailwind.config.js (hunk 4 above)
Comment (rodrigolabs) [reply platform/ui/tailwind.config.js:724]: @dannyrb just clarifying here, `group-hover` or `group-focus` is used for styling a child element when hovering over a parent element. It's a tailwind feature, but some variants like `borderColor` don't have originally group-focus and we have to add this class, like we already have for `textColor`, for example.

**Pull request merged**